docs/how-to-guides/create-multi-node-cluster.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Create a Multi-Node Cluster with k3d
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+Rancher Desktop provides a `single cluster with single node` setup, which is adequate for most local development scenarios. However, there are use cases where, the ability to create a multi node cluster or spin up multiple clusters with flexibilty to switch between clusters is required. Eventhough Rancher Desktop doesn't have in-built multi node/cluster functionality, you can use [k3d](https://k3d.io) with Rancher Desktop to accomplish the same. k3d is a lightweight wrapper to run k3s (SUSE Rancher’s minimal Kubernetes distribution, which is used by Rancher Desktop as well) in docker. k3d makes it very easy to create single- and multi-node k3s clusters in docker, e.g. for local development on Kubernetes.
+
+### Steps to spin up a multi-node cluster
+
+1. Make sure `dockerd(moby)` is selected as the Container Runtime in the `Kubernetes Settings` page.
```

Comment:
Should `dockerd(moby)` and `Kubernetes Settings` be marked as code or use bold/italic?

Reply:
Modified formatting from `code` to **bold**.